larq/callbacks_test.py

```diff
@@ -46,3 +52,58 @@ def test_quantization_logger(self):
         model.fit(x, y, batch_size=1, epochs=3, callbacks=[logger, history])
         assert history.batches == [4, 9, 14, 19] * 3
         assert history.epochs == [0, 1, 2]
+
+
+class TestHyperparameterScheduler():
+    def test_hyper_parameter_scheduler(self):
+        np.random.seed(1337)
+        (x_train, y_train), (x_test, y_test) = testing_utils.get_test_data(
+            train_samples=1000, test_samples=0, input_shape=(10,), num_classes=2
+        )
+
+        y_train = keras.utils.to_categorical(y_train)
+
+        model = lq_testing_utils.get_small_bnn_model(
+            x_train.shape[1], 20, y_train.shape[1]
+        )
+        bop_optimizer = lq.optimizers.Bop(fp_optimizer=tf.keras.optimizers.Adam(0.01))
+        model.compile(
+            loss="categorical_crossentropy",
+            optimizer=bop_optimizer,
+            metrics=["accuracy"],
+        )
+
+        scheduler = lambda x: 1.0 / (1.0 + x)
```

Comment:
I think assigning lambda functions to variables is generally discurraged in Python. Let's just go with a normal function:
```suggestion
        def scheduler(epoch):
            return 1 / (1 + epoch)
```